### ISSUE TITLE
Option to hide 🎤 voice messages button in the 📎 attachments menu

### ISSUE THREAD

**Issue by HarHarLinks** [SchildiChat/SchildiChat-android#114]
**Is your feature request related to a problem? Please describe.**
in 1.4.0 there are 2 icons in the composer: 📎 and 🎤

**Describe the solution you'd like**
Since I don't use 🎤, I would like a toggle setting to put it into the 📎 menu

**Additional context**
https://github.com/vector-im/element-android/issues/5250

**Comment by SpiritCroc** [comment]
You can hide the microphone icon by going to labs and disabling "voice messages". That will replace it with a send icon. I'd prefer to not completely hide that icon, so the layout jumps less when you start typing (and thus the "send" button appears). I can imagine that if you disable the voice icon this way because you don't often send voice messages, it may still be a nice idea to put a voice message sending possibility into the attachment area.

**Comment by HarHarLinks** [comment]
> You can hide the microphone icon by going to labs and disabling "voice messages".

yep, but then it's completely disabled, as you go on to say

> That will replace it with a send icon. I'd prefer to not completely hide that icon, so the layout jumps less when you start typing (and thus the "send" button appears).

there could reasonably be only the :paperclip: icon which gets replaced by the send button when typing. this shouldn't be too problematic because you can't start composing and then send that at the same time as any of the attachments?

**Comment by SpiritCroc** [comment]
> there could reasonably be only the paperclip icon which gets replaced by the send button when typing. this shouldn't be too problematic because you can't start composing and then send that at the same time as any of the attachments?

Hm, right, why did I not think of this :thinking:
Not sure if it's worth it though due to merge conflicts. The setting to disable the voice message button already caused quite some conflicts in the past, since Element likes to rewrite or refactor the composer area every few releases, it seems...

**Comment by HarHarLinks** [comment]
yeah understandable, depends a bit on how https://github.com/vector-im/element-android/issues/5250 goes
